### PR TITLE
Fix run_doc for led plugin

### PR DESCRIPTION
**Pull request by GiovanniVolta**
_Before you submit this PR: make sure to put all operations-related information in a wiki-note, a PR should be about code and is publicly accessible_

## What does the code in this PR do / what does it improve?

I realized that there are run_ids with more than one comment, which will cause an error as it is right now. The first comment is always from the automatic script.

I have fixed that and I have added a check for verifying that the first comments has the right format.

## Can you give a minimal working example (or illustrate with a figure)?

_Please include the following if applicable:_
  - [ ] _Update the docstring(s)_
  - [ ] _Update the documentation_
  - [ ] _Tests to check the (new) code is working as desired._
  - [ ] _Does it solve one of the open issues on github?_

### _Notes on testing_
 - _Until the automated tests pass, please mark the PR as a draft._
 - _On the XENONnT fork we test with database access, on private forks there is no database access for security considerations._

All _italic_ comments can be removed from this template.

### CODE REVIEW

**Review by dachengx — DISMISSED** [review]

**Comment by GiovanniVolta** [comment]
[Here](https://xenonnt.slack.com/archives/C016UJZ090B/p1730120125020709), @dachengx suggested changing the config in the test instead of adding a specific run_id for the led_calibration. Unfortunately, I don't know where to change it. If you prefer me to change the config, I need some help

**Comment by GiovanniVolta** [comment]
Ah, of course, the data are not stored so the test are still failing

File: straxen/plugins/led_cal/led_calibration.py [hunk 1]
```diff
@@ -56,6 +56,12 @@ class LEDCalibration(strax.Plugin):
         ),
     )
 
+    defualt_run_comments = straxen.URLConfig(
```
Comment (dachengx) [inline straxen/plugins/led_cal/led_calibration.py:59]: Please assign its type to be `list`.

**Review by dachengx — CHANGES_REQUESTED** [review]

File: straxen/plugins/led_cal/led_calibration.py (hunk 1 above)
Comment (GiovanniVolta) [reply straxen/plugins/led_cal/led_calibration.py:59]: Done

**Comment by GiovanniVolta** [comment]
@dachengx @minzhong98 @MerzJohannes @yuema137 shall we proceed here?

**Review by dachengx — APPROVED** [review]

**Comment by GiovanniVolta** [comment]
Thank you Dacheng <3

**Pull request merged**